python/lsst/ap/association/packageAlerts.py

```diff
@@ -165,6 +177,80 @@ def createDiaSourceBBox(self, bboxSize):
             bbox = geom.Extent2I(bboxSize, bboxSize)
         return bbox
 
+    def createCcdDataCutout(self, cutout, sphPoint, photoCalib):
+        """Convert a cutout into a calibrate CCDData image.
+
+        Parameters
+        ----------
+        cutout : `lsst.afw.image.ExposureF`
+            Cutout to convert.
+        sphPoint : `lsst.geom.SpherePoint`
+            Center point of DiaSource on the sky.
+        photoCalib : `lsst.afw.image.PhotoCalib`
+            Calibrate object of the image the cutout is cut from.
+
+        Returns
+        -------
+        ccdData : `astropy.nddata.CCDData`
+            CCDData object storing the calibrate information from the input
+            difference or template image.
+        """
+        cutOutMinX = cutout.getBBox().minX - 1
+        cutOutMinY = cutout.getBBox().minY - 1
+        center = cutout.getWcs().skyToPixel(sphPoint)
+        calibCutout = photoCalib.calibrateImage(cutout.getMaskedImage())
+
+        cutoutWcs = wcs.WCS(naxis=2)
+        cutoutWcs.array_shape = (cutout.getBBox().getWidth(),
+                                 cutout.getBBox().getWidth())
+        cutoutWcs.wcs.crpix = [center.x - cutOutMinX, center.y - cutOutMinY]
+        cutoutWcs.wcs.crval = [sphPoint.getRa().asDegrees(),
+                               sphPoint.getDec().asDegrees()]
```

Comment:
Isn't this crval actually for the center of the bbox?

Reply:
It's the center of the DiaSource, see above.